docs/source/doc_combine_counts.md

```diff
@@ -1,73 +1,76 @@
+
+Currently, it produces one such file that excludes sex chromosomes (for use in HATCHet), and one that includes sex chromosomes (for future use).
 
 ## Main parameters
 
 combine-counts has some main parameters; the main values of these parameters allow to deal with most of datasets, but their values can be changed or tuned to accommodate the features of special datasets.
 
 | Name | Description | Usage | Default |
 |------|-------------|-------|---------|
-| `-d`, `--diploidbaf` | Maximum expected shift from 0.5 for BAF of diploid or tetraploid clusters | The maximum shift is used to identify all the potential clusters with base states (1, 1) or (2, 2). The value depends on the variance in the data (related to noise and coverage); generally, higher variance requires a higher shift. Information provided by plot-bins can help to decide this value in special datasets. | 0.08 (other typically suggested values are 0.1-0.11 for higher variance and 0.06 for low variance) |
+| `--msr`  | Minimum SNP-covering reads for each bin | Each bin constructed by this command must have at least this many reads covering heterozygous SNPs in each sample | 5000 |
+| `--mtr`  | Minimum total reads for each bin | Each bin constructed by this command must have at least this many total reads in each sample | 5000 |
 
 ## Optional parameters
 
 | Name | Description | Usage | Default |
 |------|-------------|-------|---------|
-| `-v`, `--verbose`  | Verbose logging flag | When enabled, combine-counts outputs a verbose log of the executiong | Not used |
-| `-r`, `--disablebar` | Disabling progress-bar flag | When enabled, the output progress bar is disabled | Not used |
-| `-b`, `--normalbafs` | File of allele counts for SNPs in matched-normal sample | When provided, combine-counts attempts to correct the estimated BAF using the variance in matched-normal sample. | Not used (deprecated) |
-| `-d`, `--diploidbaf` | Maximum expected shift from 0.5 for BAF of diploid or tetraploid clusters | The maximum shift is used to identify potential potential bins with base states (1, 1) or (2, 2) whose BAF needs to be corrected. The value depends on the variance in the data (related to noise and coverage); generally, higher variance requires a higher shift. Information provided by plot-bins can help to decide this value in special datasets. | 0.08 (other typically suggested values are 0.1-0.11 for higher variance and 0.06 for low variance) |
-| `-t`, `--totalcounts` | File of total read counts | When provided, the total read counts are used to normalize the read counts from the corresponding sample | Not used (deprecated) |
-| `-m`, `--mode` | Mode used to estimate BAFs | Different modes are provided to combine to combine the allele counts of SNPs | The counts from the allele with minor count are combined (deprecated) |
+| `-j`, `--processes` | Number of parallel processes to use (default 1) |  | 1 |
```

Comment:
This is something we may want to standardize - anytime the `-j` flag is used, either default it to `1` (or `n_processors` - 1). Probably we can open an issue on this.

Reply:
I see an issue has been added for this. Closing here.